File: _posts/front/js/실험/array/myq4/1.js
const hoge = { 0: {}, 1: {}, 100: {} };

console.log(hoge);
const hoge2 = { 0: {}, 1: {}, 100: {} };

const hoge3 = { ...hoge, ...hoge2 };
console.log(hoge3);

const hoge4 = { list: [hoge, hoge2] };

console.log(hoge4);
console.log("=============");
//const foge = { 0: {}, 1: {}, 100: {} };
const foge = [{ 0: {}, 1: {}, 100: {} }];
console.log(foge);
const foge2 = [{ 0: {}, 1: {}, 100: {} }];

//const foge3 = {};
const foge3 = [];

const foge9 = [...foge, ...foge3];
console.log(foge9);

File: _posts/backend/ruby_on_rails/ruby/判定(nil,empty,blank,present)/defined/1.rb
@@hoge5 = "xx"
p defined? @@hoge5 #=> "class variable"

p defined? hoge

hoge2 = nil
p defined? hoge2

hoge3 = ""
p defined? hoge3


@hoge4 = "xx"
p defined? @hoge4 #=>  "instance-variable"

